index.html

```diff
@@ -8,18 +8,28 @@
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  <header class="header-background">
-   <nav>
-     <div  class="top-header display-flex">
-      <a class="welcome identical-to" href="#">Welcome</a>
-      <span class="identical-to">&#8801;</span>
-     </div>     
+  <header class="header-background">    
+   <nav>      
       <ul class="desktop-nav">       
         <li class="desktop-nav-list"><a class="text-color" href="#">Portfolio</a></li>
         <li class="desktop-nav-list"><a class="text-color" href="#">About</a></li>
         <li class="desktop-nav-list"><a class="text-color" href="#">Contact</a></li>        
```

Comment:
- [ ] your navigation work fine in the mobile version but it doesn't work in the desktop version , It would be better if you try to fix that

Reply:
@youmari , thanks you for your review. This point is not clear. How exactly do you mean. Can you give specific detail on what is not working on the navigation. Or screenshot

Reply:
@when I click on links on the menu they are working but when I click on the links in desktop design the links don't work they should take you to the correct path

Reply:
Hi @youmari 

Microverse instructions is beyond the change you are requesting for this particular design.

here is the list of instructions/requirements which the design already met. Desktop was not mentioned.

“In mobile, implement the following features:

When the user clicks (or taps) the hamburger button, the mobile menu appears.
When the user clicks (or taps) the close (X) button, the mobile menu disappears.
When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.”

Here is the link for full info 

https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md

Reply:
@stephnna I see your point and technically you're right and my job here is to provide better improvements, and this portfolio you're going to use to show your projects 
what if a recruiter enters your portfolio and he/she clicks on your navigations links and they aren't working what do you think is the impression that he/she will take? 
anyway I will count it as optional so the next reviewer won't ask you to do it